package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "react-dev": "webpack -d --watch",
     "server-dev": "nodemon server/index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest --watch"
```

Comment:
Do you need to be watching your tests?

Reply:
@ bill it helps to have a live test to see if it works right away and a video says they highly recommend that you should watch it always